test/test_common/environment.cc

```diff
@@ -85,6 +85,18 @@ void TestEnvironment::createParentPath(const std::string& path) {
   RELEASE_ASSERT(::system(("mkdir -p $(dirname " + path + ")").c_str()) == 0, "");
 #endif
 }
+
+void TestEnvironment::removePath(const std::string& path) {
+#ifdef __cpp_lib_experimental_filesystem
+  // We don't want to rely on rm etc. if we can avoid it, since it might not
+  // exist in some environments such as ClusterFuzz.
+  std::experimental::filesystem::remove_all(std::experimental::filesystem::path(path));
+#else
+  // No support on this system for std::experimental::filesystem.
+  RELEASE_ASSERT(::system(("rm -rf " + path).c_str()) == 0, "");
```

Comment:
I know this is test only, but this still scares me. Are we adequately isolated from all possible pilot errors here? Maybe just add a really simple `RELEASE_ASSERT` check that the first character of path isn't `.`